test/e2e/e2e.go

```diff
@@ -126,7 +126,7 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 			framework.DumpAllNamespaceInfo(c, api.NamespaceSystem)
 		}
 		framework.LogFailedContainers(api.NamespaceSystem)
-		framework.RunKubernetesServiceTestContainer(framework.TestContext.RepoRoot, api.NamespaceDefault)
+		framework.RunKubernetesServiceTestContainer(api.NamespaceDefault)
```

Comment:
if I'm reading the docs correctly, we probably want to pass `-prefix "${KUBE_ROOT}"` so it removes that from the embedded paths.